app/repositories/InvitationsCacheRepository.scala

```diff
@@ -51,26 +50,25 @@ class InvitationsCacheRepository @Inject()(
   private val encrypted: Boolean = config.getBoolean("encrypted").getOrElse(true)
   private val jsonCrypto: CryptoWithKeysFromConfig = CryptoWithKeysFromConfig(baseConfigKey = encryptionKey, config)
 
-  private def getExpireAt = DateUtils.dateTimeFromDateToMidnightOnDay(DateTime.now(DateTimeZone.UTC), 30)
-
   private case class DataEntry(
                                 inviteePsaId: String,
                                 pstr: String,
                                 data: BSONBinary,
                                 lastUpdated: DateTime,
-                                expireAt: DateTime)
+                                expireAt: Option[DateTime])
```

Comment:
It doesn't look like the expireAt field is used at all now - expireAt is assigned to the lastUpdated field. Is this right? Also, why has the field been reverted to an Option[DateTime]?

Reply:
it was part of conflicts during last merge with master, updated correctly

Reply:
OK, looks fine now